src/replication.c

```diff
@@ -4348,6 +4349,17 @@ initialize_replication()
 		return -1;
 	}
 	clock_gettime(CLOCK_MONOTONIC_COARSE, &istgt_start_time);
+	const char *s_max_wait_time = getenv("IO_MAX_WAIT_TIME");
+	int max_wait_time= 0;
+	if (s_max_wait_time != NULL)
+		max_wait_time = (int)strtol(s_max_wait_time,
+		    NULL, 10);
+	if (max_wait_time > 30) {
```

Comment:
So with this configuration, we are allowing users to decrease the response time from 60 and >30. Is this expected to happen?

Reply:
if user wants, it can happen